### ISSUE TITLE
KeysExpressionOrColumn(sample_by) exception

### ISSUE THREAD

**Issue by albertalexandrov** [xzkostyan/clickhouse-sqlalchemy#127]
It seems [here](https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/master/clickhouse_sqlalchemy/engines/mergetree.py#L35) should be unpacking with **to_list** function as it is done for **primary_key**, **order_by**, **partition_by**, **ttl**.

As you can see on screenshot below in **sample_by** there is a **list**:

![clickhouse-sqlalchemy-1](https://user-images.githubusercontent.com/23693954/111794956-d2263e80-88d7-11eb-9b7d-e255f40316bd.png)

That causes exception:

>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/clickhouse_sqlalchemy/drivers/base.py", line 829, in reflecttable
>     self._reflect_engine(connection, table.name, table)
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/clickhouse_sqlalchemy/drivers/base.py", line 841, in _reflect_engine
>     engine = engine_cls.reflect(table, **e)
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/clickhouse_sqlalchemy/engines/mergetree.py", line 224, in reflect
>     **cls._reflect_merge_tree(table, **kwargs)
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/clickhouse_sqlalchemy/engines/mergetree.py", line 201, in __init__
>     super(ReplacingMergeTree, self).__init__(*args, **kwargs)
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/clickhouse_sqlalchemy/engines/mergetree.py", line 35, in __init__
>     self.sample_by = KeysExpressionOrColumn(sample_by)
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/clickhouse_sqlalchemy/engines/base.py", line 54, in __init__
>     super(KeysExpressionOrColumn, self).__init__(*columns, **kwargs)
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 3051, in __init__
>     _to_schema_column_or_string(c) for c in columns
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 3051, in <listcomp>
>     _to_schema_column_or_string(c) for c in columns
>   File "/home/albert/Work/platform/venv3.6/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 3028, in _to_schema_column_or_string
>     raise exc.ArgumentError(msg % element)
> sqlalchemy.exc.ArgumentError: Element ['ts'] is not a string name or column element

![clickhouse-sqlalchemy-2](https://user-images.githubusercontent.com/23693954/111795503-4fea4a00-88d8-11eb-830e-40038d2c9ccf.png)

- clickhouse-sqlalchemy==0.1.6
- Python==3.6

Please fix as soon as you can. Thanks in advance.

**Comment by albertalexandrov** [comment]
Hi @xzkostyan ! 
Could you tag changes?

**Comment by xzkostyan** [comment]
Hi.

Fix will be included in new release. You can always install desired branch/commit directly from github before release:

```
pip install -I git+https://github.com/xzkostyan/clickhouse-sqlalchemy@master#egg=clickhouse-sqlalchemy
```

**Comment by albertalexandrov** [comment]
I cannot install as you suggested because we use it as dependency (in requirements.txt) for some our library and it seems setuptools (setup.py) does not support such sintax. We got error:

>   ERROR: Command errored out with exit status 1:
>      command: /home/albert/Work/FromKafkaToClickHouseInStreamingMode/venv3.8/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/home/albert/Work/data-provider/setup.py'"'"'; __file__='"'"'/home/albert/Work/data-provider/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-gvh4an2q
>          cwd: /home/albert/Work/data-provider/
>     Complete output (2 lines):
>     WARNING: The wheel package is not available.
>     error in data-provider setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Parse error at "'+https:/'": Expected stringEnd

_setupy.py_:

```
import os
import re

from setuptools import setup, find_packages


def read(file_path: str):
    return open(
        os.path.join(
            os.path.dirname(__file__),
            file_path,
        ),
    ).read()


setup(
    name="data-provider",
    use_scm_version=True,
    setup_requires=['setuptools_scm'],
    description="description",
    author="author",
    url="url",
    packages=find_packages(),
    install_requires=read('requirements.txt').splitlines()
)
```

_requirements.txt_:

git+https://github.com/xzkostyan/clickhouse-sqlalchemy@50127a01f9a296f30503c9a298375bc8b63ef0e4

Could you say please when you are planning new release?

**Comment by xzkostyan** [comment]
I have no fixed date for next release. It's not a good idea to make a new release with one-two minor fixes.

Try this solution for setuptools: https://stackoverflow.com/a/3472494

**Comment by xzkostyan** [comment]
Fixed in 0.1.7.
